pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -509,7 +509,7 @@ private static String getErrorMessage(HttpUriRequest request, CloseableHttpRespo
     StatusLine statusLine = response.getStatusLine();
     String reason;
     try {
-      reason = JsonUtils.stringToJsonNode(EntityUtils.toString(response.getEntity())).get("error").asText();
+      reason = JsonUtils.stringToJsonNode(EntityUtils.toString(response.getEntity())).get("_error").asText();
     } catch (Exception e) {
       reason = "Failed to get reason";
```

Comment:
Could we also print out some useful info like the raw output if any exception is throw when parsing the entity?

Reply:
The whole purpose of this method, `getErrorMessage` is to have a nicer log message. It defeats its purpose if we log/print out something here. With the bug fix, it should work as intended.